.changeset/rude-shrimps-arrive.md

```diff
@@ -0,0 +1,5 @@
+---
+"inngest": minor
```

Comment:
Is this `patch` instead of `minor` if it's just internal reliability improvements?
```suggestion
"inngest": patch
```